wiki/Manual.md

```diff
@@ -4253,7 +4255,7 @@ We have different classification of trigger orders:
 * an attached Stop-Loss or Take-Profit order into a primary order ([Conditional Trigger Order](#stopLoss-and-takeProfit-orders-attached-to-a-position)).
 
 
-##### Trigger Orders
+##### Trigger order
```

Comment:
trigger orders would lead users to above paragraph (which is the correct place, not here)